module/zfs/zio_compress.c

```diff
@@ -102,7 +98,8 @@ zio_compress_zeroed_cb(void *data, size_t len, void *private)
 }
 
 size_t
```

Comment:
Let's add a comment here explaining the `compress_threshold` parameter, e.g.:
`Only compress if it results in saving at least compress_threshold bytes.`